telco-hub/configuration/reference-crs-kube-compare/optional/logging/clusterLogForwarder.yaml

```diff
@@ -0,0 +1,50 @@
+---
+# ClusterLogForwarder for Telco Hub
+# Forwards audit and infrastructure logs to Kafka with hub-specific labeling
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+  annotations:
+    # Deploy after cluster logging operator is ready
+    argocd.argoproj.io/sync-wave: "10"
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    # Ignore controller-managed status differences in ArgoCD
+    argocd.argoproj.io/compare-options: IgnoreExtraneous
+spec:
```

Comment:
The majority of the spec we allow the user to specify. Only pieces explicitly noted below should be enforced (validated).

Reply:
Done.